src/main/java/uk/gov/hmcts/reform/pip/account/management/database/MediaLegalApplicationRepository.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.reform.pip.account.management.database;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import uk.gov.hmcts.reform.pip.account.management.model.MediaAndLegalApplication;
+
+import java.util.List;
+import java.util.UUID;
+
+public interface MediaLegalApplicationRepository extends JpaRepository<MediaAndLegalApplication, UUID> {
+
+    @Query(value = "SELECT * FROM media_and_legal_application WHERE status = :status", nativeQuery = true)
```

Comment:
Rather than using a native query here, we should be able to use the spring naming to do it automatically :) (Section 4.4.2 details it a bit here: https://docs.spring.io/spring-data/jpa/docs/current/reference/html/#repositories.query-methods.details)